### PR TITLE
Unify Layout and add TOC to FAQ

### PR DESCRIPTION
**Pull request by Simon-Laux**

### CODE REVIEW

File: de/blog.po [hunk 1]
```diff
@@ -21,13 +21,9 @@ msgstr ""
 msgid ""
 "---\n"
 "title: Blog\n"
-"layout: default-en\n"
+"lang: en\n"
```
Comment (ghost) [inline de/blog.po:24]: This is weird. Is this because the blog isn't translated anyway?

File: _layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+<link rel="stylesheet" href="../assets/css/styles.css" type="text/css" />
+<link rel="alternate" href="https://delta.chat/feed.xml" type="application/atom+xml" title="{{ tx.rss_news_title }}" />
+</head>
+
+<body id="top">
+
+<div id="menu">
+<ul>
+<li><a href="../en/">{{ tx.menu.home }}</a></li>
```
Comment (ghost) [inline _layouts/default.html]: If you click "Startseite" while you are on https://delta.chat/208/de/download, you will end up at https://delta.chat/208/en/. Is this intended?

File: de/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Der Messenger
-layout: default-de
+title: The messenger
+lang: de
```
Comment (ghost) [inline de/index.md:3]: Why changing a german translation back to english? Is it to be translated?

File: de/blog.po (hunk 1 above)
Comment (Simon-Laux) [reply de/blog.po:24]: its how it is, blog is only english.

File: _layouts/default.html (hunk 2 above)
Comment (Simon-Laux) [reply _layouts/default.html]: no I'll fix it.

File: de/index.md (hunk 3 above)
Comment (Simon-Laux) [reply de/index.md:3]: I didn't do that its the translation system.

**Comment by ghost** [comment]
Awesome.

**Pull request merged**